### PR TITLE
ansible: using forked version of python-jenkins until we can fix upstream

### PR DESCRIPTION
**Pull request by alfredodeza**
This is already live in prado.ceph.com for slave.yml

### CODE REVIEW

**Comment by ktdreyer** [comment]
this patch is headed upstream at https://review.openstack.org/460363

File: ansible/slave.yml [hunk 1]
```diff
@@ -218,8 +218,9 @@
 
     - name: install python-jenkins
       sudo: true
-      # https://bugs.launchpad.net/python-jenkins/+bug/1500898
-      pip: name=python-jenkins version=0.4.7
+      # HORRIBLY BROKEN. This is temporary until this lands upstream:
+      # https://github.com/ceph/python-jenkins/commit/8e018bf7d88dfc308833d195a6ebd29231a8969d
```
Comment (ktdreyer) [inline ansible/slave.yml]: Want to just link to https://review.openstack.org/460363 here?

File: ansible/slave_libvirt.yml [hunk 2]
```diff
@@ -188,8 +188,9 @@
 
     - name: install python-jenkins
       sudo: true
-      # https://bugs.launchpad.net/python-jenkins/+bug/1500898
-      pip: name=python-jenkins version=0.4.7
+      # HORRIBLY BROKEN. This is temporary until this lands upstream:
+      # https://github.com/ceph/python-jenkins/commit/8e018bf7d88dfc308833d195a6ebd29231a8969d
```
Comment (ktdreyer) [inline ansible/slave_libvirt.yml]: https://review.openstack.org/460363

File: ansible/slave.yml [hunk 3]
```diff
@@ -218,8 +218,9 @@
...
+      pip: name=git+https://github.com/ceph/python-jenkins@forked#egg=python-jenkins
```
Comment (andrewschoen) [inline ansible/slave.yml]: the branch name is actually ``patched`` I believe

File: ansible/slave_libvirt.yml [hunk 4]
```diff
@@ -188,8 +188,9 @@
...
+      pip: name=git+https://github.com/ceph/python-jenkins@forked#egg=python-jenkins
```
Comment (andrewschoen) [inline ansible/slave_libvirt.yml]: The branch name is actually ``patched``

**Comment by andrewschoen** [comment]
It looks like we missed the ``examples/slave_static.yml`` playbook here as well.

**Pull request merged**